Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM ubuntu:22.04
 
 # The GS_VERSION argument could be used like this to overwrite the default:
-# docker build --build-arg GS_VERSION=2.21.2 -t geoserver:2.21.2 .
-ARG TOMCAT_VERSION=9.0.68
-ARG GS_VERSION=2.22.0
+# docker build --build-arg GS_VERSION=2.23.0 -t geoserver:2.23.0 .
+ARG TOMCAT_VERSION=9.0.74
+ARG GS_VERSION=2.23.0
 ARG GS_DATA_PATH=./geoserver_data/
 ARG ADDITIONAL_LIBS_PATH=./additional_libs/
 ARG ADDITIONAL_FONTS_PATH=./additional_fonts/
 ARG CORS_ENABLED=false
 ARG CORS_ALLOWED_ORIGINS=*
 ARG CORS_ALLOWED_METHODS=GET,POST,PUT,DELETE,HEAD,OPTIONS
 ARG CORS_ALLOWED_HEADERS=*
+ARG WAR_ZIP_URL=https://downloads.sourceforge.net/project/geoserver/GeoServer/${GS_VERSION}/geoserver-${GS_VERSION}-war.zip
 ARG STABLE_PLUGIN_URL=https://downloads.sourceforge.net/project/geoserver/GeoServer/${GS_VERSION}/extensions
+ARG COMMUNITY_PLUGIN_URL=''
```

Comment:
Could this also have a default value, similar to `STABLE_PLUGIN_URL` ?

Reply:
For nightly builds (e.g., 2.23.x) everything should be pointing to build.geoserver.org, both stable plugins and community modules:

* https://build.geoserver.org/geoserver/2.23.x/ext-latest/
* https://build.geoserver.org/geoserver/2.23.x/community-latest/

For stable releases it gets trickier, as GS_VERSION needs a bit of string manipulation to go from 2.23.0 to 2.23.x, and then be used only for the community plugins.